classes/decision/Collector.inc.php

```diff
@@ -0,0 +1,134 @@
+<?php
+/**
+ * @file classes/decision/Collector.inc.php
```

Comment:
Did I miss something? It doesn't look like other files in `classes/` are `.php`, and I don't see any `import` statements in this file.

Reply:
I didn't see imports, just a general comment. We currently have both our custom autoloader (which looks for `.inc.php` files based on the namespace) *and* PSR-4-based autoloading support thanks to Composer. In effect this means OJS/OMP/OPS are looking for *both* `.inc.php` and `.php` files when autoloading, but before 3.4.0 gets released we'll remove our custom autoloader (once I get the opportunity to rename everything).

TL;DR: No harm in adding new classes as `.php` rather than `.inc.php`, but if you'd rather not fiddle with it, I'll get it later with the batch rename.

Reply:
Ok, I'll leave it for the batch rename but if you get that in before my merge I can rename these.